src/main/kotlin/blackjack/model/Hand.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.model
+
+class Hand() {
+    private val hold: Hold = Hold(setOf())
```

Comment:
```suggestion
    private val hold: Hold = Hold(emptySet())
```

What do you think is the difference between these two?
**Hint**: I recommend checking out the implementation of `setOf()` :)

Reply:
I guess it's because 
1. it's more readable
2. it's inefficient to call emptySet() again in set Of()

I often forget empty when I'm typing fast. I'll pay more attention!